lightstep/resource_dashboard.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
-func getUnifiedQuerySchema() map[string]*schema.Schema {
+func getUnifiedQuerySchemaMap() map[string]*schema.Schema {
```

Comment:
a bunch of functions are named with the same pattern. some return a schema while others return a map of schemas so I renamed some